docs/index.md

```diff
@@ -103,17 +100,17 @@ In addition to [generic `provider` arguments](https://www.terraform.io/docs/conf
 
 * `insecure` - (Optional) This setting allows the provider to skip TLS verification.
   Useful when targeting installation with self-signed certificates. Not recommended when
-  targeting the Elasticsearch Service (ESS).
+  targeting ESS.
 
 * `insecure` - (Optional) This setting allows the user to set a custom timeout in the
-  individual HTTP request level. Defaults to "1m" but might need to be tweaked if timeouts
+  individual HTTP request level. Defaults to 1 minute, but might need to be tweaked if timeouts
   are experienced.
 
-* `verbose` - (Optional) When set to true, it'll write a "requests.json" file in the folder
-  where terraform is executed with all outgoing HTTP requests and responses. Defaults to "false".
+* `verbose` - (Optional) When set to `true`, it writes a `requests.json` file in the folder
+  where Terraform runs with all the outgoing HTTP requests and responses. Defaults to `false`.
 
 * `verbose_credentials` - (Optional) When set with `verbose`, the contents of the Authorization
-header will not be redacted. Defaults to `"false"`.
+header is not written. Defaults to `false`.
```

Comment:
In this case by "will not be redacted", we meant to say that the contents of the Authorization header will be shown, that is to say the credentials will be visible, as opposed to seeing a `REDACTED` text.

```console
# This is what people will see
Authorization: ApiKey LASJKFHE5645LKJSDFASDASDASDASD==

# Instead of this
Authorization: ApiKey REDACTED
```

Reply:
Thanks for the clarification, Karen. If this is common terminology, then I'll revert it back to `reacted`.